masq_lib/src/logger.rs

```diff
@@ -879,6 +791,104 @@ mod tests {
         tlh.exists_log_containing("error! 42");
     }
 
+    // If this test suddenly starts failing, but only when other tests are run with it, it's probably
+    // because one or more of those other tests uses running_test(). When running_test() is active,
+    // prepare_log_recipient() won't panic properly.
+    #[test]
+    #[should_panic(expected = "Log recipient should be initiated only once")]
+    fn prepare_log_recipient_should_be_called_only_once_panic() {
+        let _guard = prepare_test_environment();
+        let ui_gateway = TestUiGateway::new(0, Arc::new(AtomicU32::new(0)), 10);
```

Comment:
I took out the ten-second time-to-live parameter, since it was always set to the same thing. But the `Arc` is asserted in every test except this one; I don't want to change the signature of the constructor for one test.